cypress/e2mock/teams-permissions.cy.js

```diff
@@ -0,0 +1,26 @@
+import selectors from '../support/selectors'
+
+const fixtures = {
+  errorMessage: {
+    detail: 'You do not have permission to perform this action (teams management).'
+  }
+}
+describe('Tams Permissions Spec', () => {
+  beforeEach(() => {
+    cy.login()
+    cy.openProduct('Teams Permissions')
+  })
+
+  it('should block users without acess of entering in the edit page', function () {
+    //act
+    cy.get(selectors.list.searchField).type('Default Team')
+    cy.intercept('GET', '/api/v4/iam/teams/*', { statusCode: 403, body: fixtures.errorMessage }).as(
```

Comment:
use cy.intercept, using only the specific  service that needs to be intercepted cypress in the teams collection.

Reply:
o asterisco na rota do intercept foi utilizado para que não seja necessário especificar o id do team que esta sendo requisitado